server/routes/visualization.js

```diff
@@ -33,6 +36,18 @@ router.get('/getAll', async (req, res) => {
         res.status(500).json({message: error.message})
     }
 })
+
+// Get by Title Method
+router.get('/getOne/:title', async (req, res) => {
```

Comment:
Same thing as above with categories, can just name this / and then use url params to distinguish that we're getting by title.